kart/raster/v1.py

```diff
@@ -163,3 +195,135 @@ def diff_to_working_copy(
         ds_diff["tile"] = tile_diff
 
         return ds_diff
+
+    def _non_pam_wc_paths(self, tile_and_pam_paths):
+        """
+        Given a list of dirty paths, relative to the workdir -
+        return the paths of the tiles (relative to the workdir) that have been affected
+        either by editing directly or by editing their PAM files.
+        """
+        result = set()
+        for path in tile_and_pam_paths:
+            if path.lower().endswith(".aux.xml"):
+                tile_paths = find_similar_files_case_insensitive(
+                    self._workdir_path(path[:-8])
```

Comment:
magic 8 is going to bite us when we inevitably add another type of sidecar file. maybe use `len(PAM_SUFFIX)` and then at least it's greppable

Reply:
Done